blockly/blocks/base.js

```diff
@@ -227,3 +227,64 @@ Blockly.Blocks['serial_print'] = {
     this.setTooltip('Prints data to the console/serial port as human-readable ASCII text.');
   }
 };
+
+Blockly.Blocks['lcd_setup'] = {
+  init: function() {
+    this.setColour (257);
+    this.appendDummyInput("")
+      .appendField("Setup LCD")
+      .appendField(new Blockly.FieldImage("https://c1.staticflickr.com/9/8305/7804378078_4cca315e83.jpg", 64, 64))
```

Comment:
It's a bit hard to notice what the image is when it is reduced to 64px, maybe a more generic dual line LCD screen image could be easier to recognise? I do like the fact that it is different from the Grove LCD image though, so that it cannot be easily confused with each other.